packages/app-desktop/gui/MainScreen/commands/togglePerFolderSortOrder.ts

```diff
@@ -0,0 +1,18 @@
+import { CommandContext, CommandDeclaration, CommandRuntime } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+import PerFolderSortOrderService from '../../../services/sortOrder/PerFolderSortOrderService';
+
+export const declaration: CommandDeclaration = {
+	name: 'togglePerFolderSortOrder',
+	label: () => _('Toggle per-folder sort order'),
```

Comment:
> But also, when the per-notebook option is not enabled on the notebook, this is actually like a global setting that applies not, per-notebook, but to all notebooks, is that correct?

When `notes.perFolderSortOrderEnabled` is false, this command has no effect, and it is as expected.

> I don't think we can convey all that info in this tooltip, so simply "Toggle sort order" would be sufficient (it's implied it's for the notebook).

Since this feature is not very common, I'm afraid that too brevity introduces misunderstanding such as 'toggle sort order field (title/update time/create time/custom)'. I agree that "Toggle per-notebook sort order" is too long. How about "Toggle own sort order"?